ckanext/qa/i18n/en_AU/LC_MESSAGES/ckanext-qa.po

```diff
@@ -0,0 +1,250 @@
+msgid "This URL last worked on: %s."
+msgstr ""
+
+#: ckanext/qa/tasks.py:311
+msgid "This was the first attempt."
+msgstr ""
+
+#: ckanext/qa/tasks.py:313
+#, python-format
+msgid "Tried %s times since %s."
+msgstr ""
+
+#: ckanext/qa/tasks.py:319
+msgid "This URL has not worked in the history of this tool."
+msgstr ""
+
+#: ckanext/qa/tasks.py:355 ckanext/qa/tasks.py:386
+msgid "This file had not been downloaded at the time of scoring it."
+msgstr ""
+
+#: ckanext/qa/tasks.py:360
+#, python-format
+msgid "Cache filepath does not exist: \"%s\"."
+msgstr ""
+
+#: ckanext/qa/tasks.py:368
+#, python-format
+msgid ""
+"Content of file appeared to be format \"%s\" which receives openness "
+"score: %s."
+msgstr ""
+
+#: ckanext/qa/tasks.py:372
+msgid "The format of the file was not recognized from its contents."
+msgstr ""
+
+#: ckanext/qa/tasks.py:377
+msgid "File was not downloaded deliberately"
+msgstr ""
+
+#: ckanext/qa/tasks.py:378 ckanext/qa/tasks.py:383
+msgid "Using other methods to determine file openness."
+msgstr ""
+
+#: ckanext/qa/tasks.py:382
+msgid "A system error occurred during downloading this file"
+msgstr ""
+
+#: ckanext/qa/tasks.py:403
+msgid "Could not determine a file extension in the URL."
+msgstr ""
+
+#: ckanext/qa/tasks.py:410
+#, python-format
+msgid "URL extension \"%s\" relates to format \"%s\" and receives score: %s."
+msgstr ""
+
+#: ckanext/qa/tasks.py:414
+#, python-format
+msgid ""
+"URL extension \"%s\" relates to format \"%s\" but a score for that format"
+" is not configured, so giving it default score %s."
+msgstr ""
+
+#: ckanext/qa/tasks.py:418
+#, python-format
+msgid "URL extension \"%s\" is an unknown format."
+msgstr ""
+
+#: ckanext/qa/tasks.py:455
+msgid "Format field is blank."
+msgstr ""
+
+#: ckanext/qa/tasks.py:460
+#, python-format
+msgid "Format field \"%s\" does not correspond to a known format."
+msgstr ""
+
+#: ckanext/qa/tasks.py:463
+#, python-format
+msgid "Format field \"%s\" receives score: %s."
+msgstr ""
+
+#: ckanext/qa/templates/qa/openness_stars.html:23
+#: ckanext/qa/templates/snippets/organization.html:9
+msgid "Openness"
+msgstr ""
+
+#: ckanext/qa/templates/qa/openness_stars.html:25
+#: ckanext/qa/templates/qa/openness_stars_brief.html:23
+msgid "Checked"
+msgstr ""
+
+#: ckanext/qa/templates/qa/openness_stars.html:27
+#: ckanext/qa/templates/qa/openness_stars_brief.html:26
+msgid "No openness information yet."
+msgstr ""
+
+#: ckanext/qa/templates/qa/stars.html:24
+msgid "{0} out of {1}"
+msgstr ""
+
+#: ckanext/qa/templates/qa/stars_explained.html:7
+msgid "Linked data - data URIs and linked to other data (e.g. RDF)"
+msgstr ""
+
+#: ckanext/qa/templates/qa/stars_explained.html:9
+msgid "Linkable data - served at URIs (e.g. RDF)"
+msgstr ""
+
+#: ckanext/qa/templates/qa/stars_explained.html:11
+msgid "Structured data in open format (e.g. CSV)"
+msgstr ""
+
+#: ckanext/qa/templates/qa/stars_explained.html:13
+msgid "Structured data but proprietry format (e.g. Excel)"
```

Comment:
Misspelled "proprietary".

Reply:
@ThrawnCA Spelling mistake in original code https://github.com/ckan/ckanext-qa/blob/master/ckanext/qa/templates/qa/stars_explained.html#L13 but this template is not rendered as it is overridden in the data-qld-theme with different tick explanations with the correct spelling https://github.com/qld-gov-au/ckanext-data-qld-theme/pull/155/files#diff-1cbcd41990568d29a49c1554a3abc314R14